include/struct_defs/struct_0207CDEC.h

```diff
@@ -4,6 +4,6 @@
 typedef struct {
```

Comment:
Structs must be typedef'd, and have the same name as the typedef, i.e.

`typedef struct ItemSlot {
    u16 unk_00;
    u16 unk_02;
} ItemSlot;`